indexing-service/src/main/java/io/druid/indexing/worker/WorkerTaskMonitor.java

```diff
@@ -442,6 +451,9 @@ public void handle()
               )
           );
         }
+        catch (InterruptedException e) {
+          throw e;
```

Comment:
same thing about suggesting that the interrupt status of the thread is preserved after throw.